### DISCUSSION TITLE
Writing point cloud with scalar

### DISCUSSION THREAD

**Discussion by pascalisfort** [isl-org/Open3D#5063] [Q&A]
Hello, I interested to write a point cloud to ply format with o3d, everything works well when I use it to save only XYZ coordinate, 

but is it possible to add scalars on the ply file, other than the colors and the normals?

Per example I have a point cloud with color, normals, intensity and near infrared value for each point. I used "plyfile" to write all of this scalars on a ply file, it works well for a small  point cloud but for a bigger point cloud it not works (memory error). 

Thank you for your help 
Pascal

**Comment by brleme** [comment]
hey! did you get any idea about it?
